empirical_photometry.ipynb

```diff
@@ -0,0 +1,467 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import tynt\n",
+    "from synphot.spectrum import SpectralElement\n",
+    "\n",
+    "# we leave out the u and z bands because they don't overlap with the spectrum\n",
+    "sdss_bands = ['g', 'r', 'i']\n",
+    "\n",
+    "f = tynt.FilterGenerator()\n",
+    "\n",
+    "bandpasses = {}\n",
+    "for band in sdss_bands:\n",
+    "    filt = f.download_true_transmittance('SLOAN/SDSS.' + band)\n",
+    "    bp = SpectralElement(Empirical1D,\n",
+    "                         points=filt.wavelength,\n",
+    "                         lookup_table=filt.transmittance)\n",
+    "    bandpasses[band] = bp\n",
+    "\n",
+    "    plt.plot(bp.waveset, bp(bp.waveset), label=band)\n",
+    "    \n",
+    "# plot the normalized spectrum on top to see how it overlaps with the filters\n",
+    "waveset = spectrum.waveset\n",
+    "plt.plot(waveset, spectrum(waveset) / np.median(spectrum(waveset)) / 20)\n",
+    "\n",
+    "plt.ylim(0, 0.5)\n",
+    "plt.legend(loc='upper right')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also construct a bandpass from a file using <tt>synphot</tt>'s <a href=\"https://synphot.readthedocs.io/en/latest/api/synphot.spectrum.SpectralElement.html#synphot.spectrum.SpectralElement.from_file\">SpectralElement.from_file()</a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"observe\"></a>\n",
+    "### 4. Model the observation with `synphot.observation`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To convolve the spectrum with the filters, use <a href=\"https://synphot.readthedocs.io/en/latest/synphot/observation.html\">`synphot.observation`</a>:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: Source spectrum will be extrapolated (at constant value for empirical model). [synphot.observation]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from synphot.observation import Observation\n",
+    "\n",
+    "# zero point flux of sdss system\n",
+    "zero_point_star_equiv = u.zero_point_flux(3631.1 * u.Jy)\n",
+    "\n",
+    "fluxes = {}\n",
+    "for band in sdss_bands:\n",
+    "    observation = Observation(spectrum, bandpasses[band], force='extrap')\n",
+    "    flux = observation.effstim('Jy')\n",
```

Comment:
It'd be great if you introduced `effstim` before you used it. When I first encountered it, for example, I wasn't sure what it abbreviated, and what calculation it was doing under-the-hood.

Reply:
It is documented at https://synphot.readthedocs.io/en/latest/synphot/formulae.html#effective-stimulus . FYI.